scripts/astrokat-cals.py

```diff
@@ -397,16 +391,28 @@ def table_line(datetime,  # ephem.Date object
                ):
     """Construct a line of target information to display on command line output.
 
-    @param datetime: date and time (ephem.Date object) for calculating target information
-    @param horizon: minimum pointing angle in degrees
-    @param target: target from katpoint.Catalogue object
-    @param sep_angle: [optional] separation angle in degrees as float
-    @param cal_limit: [optional] maximum separation angle between target and calibrator
-    @param sol_limit: [optional] minimum separation angle between target and Sun
-    @param lst: [optional] display times in LST rather than UTC
-    @param notes: [optional] user provided extra information
-
-    @return: <name> <risetime UTC> <settime UTC> <Separation> <Notes>
+    Parameters
+    ----------
+        datetime: date and time
+            ephem.Date object for calculating target information
+        horizon: float
+            minimum pointing angle in degrees
+        target: str
```

Comment:
Order of params in docstring doesn't match actual params for the function.  Fix docstring.